src/main/java/de/siegmar/logbackgelf/GelfEncoder.java

```diff
@@ -286,6 +291,14 @@ public void addStaticField(final String staticField) {
         addStaticField(split[0].trim(), split[1].trim());
     }
 
+    public int getMaxShortMessageLength() {
+        return maxShortMessageLength;
+    }
+
+    public void setMaxShortMessageLength(final int maxShortMessageLength) {
```

Comment:
Please extend [README.md](https://github.com/osiegmar/logback-gelf/blob/develop/README.md) by this new property.